### ISSUE TITLE
Error deprecated function in Unity 2019.3

### ISSUE THREAD

**Issue by kyon-bll** [line/line-sdk-unity#21]
## What did you do?

Open cloned this repo By Unity 2019.3.
below error happend.
```
Assets/LineSDK/Editor/CocoaPods/CocoaPodsInstalling.cs(77,51): error CS0619: 'PBXProject.GetUnityTargetName()' is obsolete: 'This function is deprecated. There are two targets now, call GetUnityMainTargetGuid() - for app or GetUnityFrameworkTargetGuid() - for source/plugins to get Guid instead of calling to TargetGuidByName(GetUnityTargetName()).'
```

## Your environment?

Unity 2019.3.7f1
LINE SDK 1.1.1

**Comment by onevcat** [comment]
Maybe duplicated to #19, and should be fixed in #20

**Comment by kyon-bll** [comment]
thanks!

**Comment by onevcat** [comment]
Should be fixed in #20. I am going to close this issue and release a patch for it. Thank you for reporting it!

**Comment by onevcat** [comment]
[Version 1.1.2](https://github.com/line/line-sdk-unity/releases/tag/1.1.2) was released to fix it.
